module/rnaseq_de_student-guide.qmd

```diff
@@ -0,0 +1,257 @@
+```
+
+
+Directly underneath the “ontology” section is information about when the gene is expressed during development. You can learn more information by clicking on the blue squares, or by clicking on the links in the upper right-hand corner. These links will take you to other websites.
+
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Directly underneath the “ontology” section is information about when the gene is expressed during development. You can learn more information by clicking on the blue squares, or by clicking on the links in the upper right-hand corner. These links will take you to other websites.", out.width = '100%'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1vikcyml6uw-_DaC1yFpS-UWoWNEGxFukMB_eCi42BqY/edit?slide=id.g346083d4dc0_0_18#slide=id.g346083d4dc0_0_18")
+```
+
+
+# Activity
+A separate Rmd file has been created for this activity. You can download it [here](). Once it’s downloaded, you should open it using RStudio.
+
+## Choose your dataset
+
+You'll first want to decide which dataset(s) you'll use for your exploration. 
+
+### Gut-brain axis: Autism Spectrum Disorder in mice
+
+[Full gene expression data; no comparisons, just counts](https://genomicseducation.org/data/mouse_autism_counts.csv)
+
+**Comparing gene expression between ASD and control mice**
+
+[Both brain regions](https://genomicseducation.org/data/mouse_autism_deseq_autismVcontrol.csv)
+
+[Prefrontal cortex only](https://genomicseducation.org/data/mouse_autism_deseq_autismVcontrol_in_prefrontalcortex.csv)
+
+[Striatum only](https://genomicseducation.org/data/mouse_autism_deseq_autismVcontrol_in_striatum.csv)
+
+**Comparing gene expression between prefrontal cortex and striatum**
+
+[All mice](https://genomicseducation.org/data/mouse_autism_deseq_tissuetype.csv)
+
+[Only ASD mice](https://genomicseducation.org/data/mouse_autism_deseq_tissuetype_in_ASDmice.csv)
+
+[Only control mice](https://genomicseducation.org/data/mouse_autism_deseq_tissuetype_in_controlmice.csv)
+
+
+## Analysis
+
+Before starting this activity, you should have completed the miniCURE activities and identified genes of interest for your individual project.
+Remember, the basic steps of your analysis are:
+
+1. Examine expression of your  gene of interest expressed in this dataset.
+1. Explore whether your gene of interest is differentially expressed in different experimental groups.
+1. Characterize the genes that are differentially expressed between the experimental groups based on their molecular activity.
+
+### Loading the tidyverse package and the data
+
+All the datasets for this activity can be loaded into R from their URL. In order to do this, we will use some code that has been saved in an R package called `tidyverse`. In order to use this code, we will need to install and open `tidyverse` using the R code chunk below.
+
+`install.packages` retrieves and installs the package
```

Comment:
bulletize?

Reply:
Changed so that it's explicit what students should type into the console